crates/core/src/compiler/transactions.rs

```diff
@@ -28,6 +25,10 @@ pub struct CompiledTransactionsDatalake {
     pub transaction_receipts: Vec<TransactionReceipt>,
     /// MMR meta data related to the headers
     pub mmr_meta: MMRMeta,
+    /// Last proof of the tx
+    pub last_tx_markers: Vec<Transaction>,
```

Comment:
Couldnt confirm this is working, as we are still ironing out the last edge cases of mpt verify